spec/features/borrower_creates_loan_request.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.feature "borrower creates loan request" do
+  let(:borrower) { User.create(name: "Richard",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.